### PR TITLE
Shoshi/home page conponent

### PR DESCRIPTION
**Pull request by Shoshana-Yudelevitz**
שיניתי את העמוד home page conponent שלא יהיה קשור לעניין הפרופילים...

### CODE REVIEW

**Review by MiryamGabay — COMMENTED** [review]
home page

File: client-side/src/styles/homePage.scss [hunk 1]
```diff
@@ -0,0 +1,130 @@
+@import url('https://fonts.googleapis.com/css2?family=Nanum+Gothic&display=swap');
+@import '../stories/variables.scss';
+
+$primary-color: rgb(103, 252, 210);
+$black-color: #000000;
+$white-color: #ffffff;
```
Comment (e171271) [inline client-side/src/styles/homePage.scss:6]: variables should be declared in variables file

File: client-side/src/styles/homePage.scss [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+$hover-color: rgba(103, 252, 210, 0.8);
+
+.homepage {
+  font-family: 'Nanum Gothic', sans-serif;
```
Comment (e171271) [inline client-side/src/styles/homePage.scss:10]: font-family should come from variables file

File: client-side/src/styles/homePage.scss [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+  color: $primary-color;
+  border: 2px solid $primary-color;
+  padding: 1.5rem 3rem;
+  text-decoration: none;
+  font-size: 1.25rem;
+  border-radius: 8px;
+  transition: background-color 0.3s, color 0.3s;
+  text-align: center;
+  display: inline-block;
+  min-width: 200px;
```
Comment (e171271) [inline client-side/src/styles/homePage.scss:32]: use % instead of px

File: client-side/src/styles/homePage.scss
Comment (e171271) [inline client-side/src/styles/homePage.scss:1]: please use scss structure, currently your file looks like css file

**Review by e171271 — CHANGES_REQUESTED** [review]

File: client-side/src/styles/homePage.scss
Comment (e171271) [reply client-side/src/styles/homePage.scss:1]: yiu should have one primary class (maybe the homePage) and all the styles should come inside

File: client-side/src/components/homePageComponent.jsx [hunk 4]
```diff
@@ -0,0 +1,149 @@
+
+// import React from 'react';
```
Comment (EA171) [inline client-side/src/components/homePageComponent.jsx:2]: please remove all the comments

File: client-side/src/styles/homePage.scss [hunk 5]
```diff
@@ -0,0 +1,266 @@
+
+// @import url('https://fonts.googleapis.com/css2?family=Nanum+Gothic&display=swap');
+// @import '../stories/variables.scss';
+
+
+// .homepage {
+//   font-family: $font-family-primary;
+//   margin: 0;
+//   padding: 0;
+//   color: $black-color;
```
Comment (EA171) [inline client-side/src/styles/homePage.scss:10]: please remove all the comments

**Review by EA171 — CHANGES_REQUESTED** [review]

**Review by e171271 — APPROVED** [review]

**Review by EA171 — APPROVED** [review]

**Pull request merged**